package.json

```diff
@@ -6,7 +6,7 @@
     "test": "lerna exec npm t",
     "build": "lerna run build",
     "docs": "./node_modules/.bin/typedoc",
-    "publish:next": "lerna publish --no-push --conventional-prerelease --dist-tag next",
+    "publish:release-client": "lerna publish --no-push --conventional-prerelease --dist-tag next --preid rc",
```

Comment:
RC usually refers to "release candidate", I've never actually heard the term "release client" before...

Reply:
my bad!